examples/v0/write.py

```diff
@@ -1,3 +1,4 @@
+# v0 API is deprecated
```

Comment:
Does this work for module docs? I've always seen/used the triple quotes here.

Reply:
Changed these to docstrings. Originally intended these to just be notes to the user but might as well make them follow convention.